requirements.txt

```diff
@@ -6,7 +6,7 @@ docutils>=0.10
 -e git://github.com/boto/botocore.git@develop#egg=botocore
 -e git://github.com/boto/s3transfer.git@develop#egg=s3transfer
 -e git://github.com/boto/jmespath.git@develop#egg=jmespath
-nose==1.3.0
+nose==1.3.7
```

Comment:
Um, im pretty sure it was the bug where nose always failed trying to output Unicode that I mentioned in the PR body.